pyaerocom/data/aliases.ini

```diff
@@ -36,7 +36,7 @@ emidust = emidu
 loaddust = loaddu
 wetdust = wetdu
 drydust = drydu
-sconcdust = sconcdu
```

Comment:
sconcdust is an old aerocom phase 2 variable name and should not be used in pyaerocom. Use concdust instead.